omc3/model/model_creators/lhc_model_creator.py

```diff
@@ -82,6 +85,13 @@ def get_correction_check_script(
 
     @classmethod
     def prepare_run(cls, accel: Lhc) -> None:
+        print(f"year: {accel.year}")
```

Comment:
replace with LOGGER.info or LOGGER debug

Reply:
should probably be removed